### ISSUE TITLE
Unable to load DLL 'minifier' or one of its dependencies

### ISSUE THREAD

**Issue by SebastianKunz** [JKamsker/NMinify#1]
Unhandled exception. System.DllNotFoundException: Unable to load DLL 'minifier' or one of its dependencies: The specified module could not be found. (0x8007007E)

How to reproduce:
Csproj
```
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <OutputType>Exe</OutputType>
        <TargetFramework>net8.0</TargetFramework>
        <ImplicitUsings>enable</ImplicitUsings>
        <Nullable>enable</Nullable>
    </PropertyGroup>

    <ItemGroup>
      <PackageReference Include="NMinify" Version="0.0.2" />
    </ItemGroup>

</Project>
```

Program.cs
```
using NMinify;

var text = "<span style=\"color:#ff0000;\" class=\"text\">Some  text</span>";
var minifier = new Minifier();
var minified = minifier.MinifyString(MinifierMediaType.Html, text);


Console.WriteLine(text); // Output: <span style="color:#ff0000;" class="text">Some  text</span>
Console.WriteLine(minified); // Output: <span style=color:red class=text>Some text</span>
```

**Comment by SebastianKunz** [comment]
Also confirmed that when cloning the repo and running `NMinify.ConsoleTest` I run into the same exception.

**Comment by JKamsker** [comment]
Hey, thank you for pointing it out!
I have not forgotten it and will get to it asap :)

**Comment by JKamsker** [comment]
The problem was, that the ``LibraryResolver`` did not look in the correct folder for the native libraries, this has been fixed in 0.0.4.
The reason why the Test project did not work is, that the native libraries have to be build from scratch before, somehow the buildscripts now produce incompatible binaries. I only got the working ones from the 0.0.2 nupkg.
Definetely something i have to look into the future is to fix the go build ™️
